### PR TITLE
Krati Agarwal Update

### PR DESCRIPTION
**Pull request by agarkra**

### CODE REVIEW

**Review by akshatvg — CHANGES_REQUESTED** [review]
Hey!

The image link is invalid. Can you check it up?

**Comment by ATechnoHazard** [comment]
To make it clear: we require an image of your website. The URL should be raw, not a shortener like the one you've provided. 

@akshatvg we should make it clear in the readme about what sort of image and URL we require.

File: users.json [hunk 1]
```diff
@@ -34,4 +34,9 @@
         "link": "https://tanut2539.github.io/portfolio",
         "image": "https://i.ibb.co/RTh56dk/tanut.png"
     }
-]
\ No newline at end of file
+        {
+        "name": "Krati Agarwal",
+        "link": "https://agarkra.github.io/kratiagarwal.github.io/",
+        "image": "https://ibb.co/tp8SDjv"
+    }
```
Comment (ATechnoHazard) [inline users.json:41]: This should be a raw URL pointing to an image of your website.

**Review by ATechnoHazard — CHANGES_REQUESTED** [review]

**Comment by akshatvg** [comment]
@ATechnoHazard will do.

File: users.json (hunk 1 above)
Comment (akshatvg) [reply users.json:41]: @agarkra please use this image link instead: https://i.ibb.co/RP1fVV1/krati.png

**Comment by akshatvg** [comment]
@agarkra please use the [new image link](https://i.ibb.co/RP1fVV1/krati.png) and fix conflicts.

**Review by akshatvg — APPROVED** [review]

**Comment by akshatvg** [comment]
@agarkra please fix the conflicts now.

**Pull request merged**